build.gradle.kts

```diff
@@ -126,18 +164,23 @@ tasks.register("createDeb") {
   }
 }
 
+/*
+ * Task: createRpm
+ *
+ * Creates the Linux RPM package file (RedHat and derrivatives).
+ */
 tasks.register("createRpm") {
   group = "build"
-  description = "Makes the Linux platform specific packages"
+  description = "Makes RPM Linux package file."
```

Comment:
`RPM package` is a kind of file, so the "file" at the end of the line is redundant.